# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ipmiutil/string_h.patch
# Copyright (C) 2007 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./util/ipmilanplus.c.orig	2007-04-06 23:26:06.000000000 +0200
+++ ./util/ipmilanplus.c	2007-04-06 23:26:30.000000000 +0200
@@ -8,6 +8,7 @@
  */
 #include <stdio.h>
 #include <stdarg.h>
+#include <string.h>
 
 // #define DEBUG  1
 #ifndef HAVE_LANPLUS
